web/client/translations/data.en-US.json

```
@@ -1603,6 +1603,7 @@
             "featureClose": "Are you sure to close the feature grid?",
             "delete": "Do you confirm to delete {count} features?",
             "missingGeometry": "Missing geometry",
+            "zoomObject": "Zoom on object",
```

hi, I like more this
```suggestion
            "zoomObject": "Zoom to feature",
```

---

Hi @MV88, 
sure I can change it, how about we use the word "geometry"? 
Feature is mostly related to functionality and when the element could not be loaded the tooltip goes by "Missing geometry", in that way we can keep it consistent.

---

Feature term here can be used in different ways:
- a functionality of the app
- the object which contains properties and geometry, a Feature JSON object for example

in GIS apps the Feature is mostly referred to the second example

although when you miss the geometry of a feature you have the tooltip "missing geometry"
having "zoom to geometry" can be ok, but for example also [qgis uses zoom to feature](https://docs.qgis.org/2.14/en/docs/user_manual/working_with_vector/attribute_table.html#zoom-to-feature) as naming

can you please make the amendments as suggested?

thank you so much for the contribution, though ;)

---

thanks for the explanation @MV88 
I am trying to use the texts used by QGIS but unfortunetely some of the translations are missing, for the case of Italian, the google translation of "Zoom to feature" would be "zoom in funzione", just double checking that it would be appropiate for this case as I don't know Italian

---

@diegovargasg happy to help
for italian you can clone from this text
"identifyZoomToFeature": "Zoom alla feature",


for other languages is enough to add locales for the main 5 ones that we support
- it
- en
- es
- de
- fr

feel free to use google translator for the others if you have time :)